unexpand/src/tab_stops.rs

```diff
@@ -0,0 +1,210 @@
+#[derive(Debug)]
+pub struct TabStops {
+    offset:    Option<usize>,
+    repetable: Option<usize>,
+    positions: Vec<usize>,
+}
+
+const ARG_PARSE_MSG: &str = "unexpand: error parsing arguments";
+
+impl TabStops {
+    pub fn new(tabs_str: Option<&str>) -> Result<TabStops, &str> {
```

Comment:
I'm not sure, but I have a feeling that `new` should always return the structure instance.
Maybe I should rename this method...

Reply:
Nah, that is ok in cases where the struct cannot be safely constructed  without check, as example, checkout `CString::new()` on `std`